huaweicloud/services/acceptance/eip/resource_huaweicloud_eip_associate_test.go

```diff
@@ -94,10 +94,10 @@ func TestAccEIPAssociate_port(t *testing.T) {
 func TestAccEIPAssociate_compatible(t *testing.T) {
 	var eip eips.PublicIp
 	rName := acceptance.RandomAccResourceName()
-	associateName := "huaweicloud_networking_eip_associate.test"
+	associateName := "huaweicloud_vpc_eip_associate.test"
```

Comment:
the test case using `huaweicloud_networking_eip_associate` to test the compatibility, please keep it as is.

Reply:
reverted